src/browser/ui/dom/components/ReactDOMInput.js

```diff
@@ -66,7 +67,7 @@ var ReactDOMInput = ReactClass.createClass({
 
   render: function() {
     // Clone `this.props` so we don't mutate the input.
```

Comment:
It may be useful to pull that comment out of `filterDisabledEvents` and place it where the `Clone this.props` comment existed.